import "nbs_status.proto";

option java_package = "com.nextev.cvs_proto.protobuf.events";
option java_outer_classname = "NBSStatusChangeMessage";

message NBSStatusChangeEvent {
    optional string id = 1; // unique id, VIN for vehicle data
    optional int32 version = 2; // message protocol version
    optional uint64 sample_ts = 3; // sample time
    optional NBSStatus nbs_status = 4;
}
